src/migrate/index.js

```diff
@@ -18,6 +18,8 @@ export default class Migrator {
   constructor(knex) {
     this.knex   = knex
     this.config = this.setConfig(knex.client.config.migrations);
+    this.LockError = function() {}
```

Comment:
`knex` includes `inherits` library and its been used in most of the places: 

e.g. 
```js
var inherits       = require('inherits')
var Client         = require('../../client')

function Client_PG(config) {
  Client.apply(this, arguments)
}
inherits(Client_PG, Client)
```